CHANGELOG.md

```diff
@@ -1,36 +1,57 @@
 # Changelog
 
 ## 2.0.0 (Unreleased)
-### Breaking Changes
-* Use [AWS-LC](https://github.com/awslabs/aws-lc/) as the backing native cryptography library for ACCP
+
+### Overview
+This is a new major release of ACCP. We provide build artifacts for Linux-x86
+and Linux-aarch64, which can be accessed either from the release section on
+Github, or via Maven Central.
+
+This version uses [AWS-LC](https://github.com/awslabs/aws-lc/) instead of
+OpenSSL (version 1.1.1j) as the underlying cryptographic library. The switch
+provides advantages over the previous version of ACCP, such as improved
+performance due to optimized assembly implementations of some cryptographic
+algorithms in AWS-LC. These optimizations are beneficial for AWS Graviton users
+as well as x86 based platforms; moreover, rigorous testing and formal
+verification in AWS-LC’s development lifecycle reduces the risk of security
+vulnerabilities.
+
+This version is not backward compatible and the differences may affect your
+application. Some major features, such as non-EC DSA and non-EC DH key exchange
+algorithms, are removed. Other minor changes include, the implementation of the
+SecureRandom relies on AWS-LC’s DRBG and the name is changed from
+`NIST800-90A/AES-CTR-256` to `LibCryptoRng`.
+
+
+This is a major release that includes some breaking changes. ACCP has switched to using [AWS-LC](https://github.com/awslabs/aws-lc/) instead of OpenSSL as the backing native crypto engine. This transition has improved the performance of ACCP. We have tried to keep the breaking changes minimal, but they have been deemed necessary. [Optimized assembly implementation of algorithms and the usage of formal verification in AWS-LC](https://github.com/awslabs/aws-lc/blob/main/README.md) are among the reasons for ACCP to switch from OpenSSL to AWS-LC. Some of these examples include dropping the support for non-EC DSA and DH key exchange algorithms; moreover, AWS-LC and OpenSSL are not 100% compatible. We have tried to keep the incompatibilities hidden from ACCP users, and we will deal with such scenarios case by case in the future.
```

Comment:
given the second paragraph (L10-L17), this paragraph (L26) seems redundant.

Reply:
I''ll fix it in a follow up. Thanks.